feature-libs/product-configurator/rulebased/components/overview-sidebar/configurator-overview-sidebar.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="cx-sticky-menu">
+  <div class="cx-menu-bar">
```

Comment:
The enclosing div needs to get the role tablist
```suggestion
  <div class="cx-menu-bar" role="tablist">
```